app/registration/views/visit-provider.html

```diff
@@ -0,0 +1,80 @@
+<div class="modal fade" id="visitProviderModal" tabindex="-1" role="dialog" aria-labelledby="visitProviderModalLabel">
```

Comment:
This dialog has no action items yet? was it supposed to loop through a collection of providers? all rows are uniform and NOT clickable